src/app/views/pages/home/Tamil.html

```diff
@@ -1,357 +1,483 @@
 <!DOCTYPE html>
```

Comment:
@sagarkoshti1990 is this change required?

Reply:
it was just a format change. i have reverted it.